src/Game.cpp

```diff
@@ -88,7 +250,8 @@ int sub_401190() {
             sub_612EB0("movie", 0, &var4);
             glo_6C2E88.sub_53ACB0(var4);
             t = 15;
-        } else if (sub_612E10("skipintro") || glo_6C2C64 != 0) t = 11;
+        }
+        else if (sub_612E10("skipintro") || glo_6C2C64 != 0) t = 11;
```

Comment:
It's just visual studios auto formatter. When I messed up my tabs it was the quickest way to switch it